website/docs/tooltip.md

```diff
@@ -252,3 +253,24 @@ useEffect(() => {
   tooltipRef.current.toggleTooltip();
 }, []);
 ```
+
+---
+
+### `closeOnlyOnBackdropPress`
+
+Flag to determine whether to disable auto hiding of tooltip when touching/scrolling anywhere inside the active tooltip popover container.
+
+- When `true`, Tooltip closes only when overlay backdrop is pressed (or) highlighted tooltip button is pressed.
+
+```js
```

Comment:
You can remove this js example. I think it's already apparent how to use this prop.

Reply:
Done :+1: